tests/functional/conftest.py

```diff
@@ -5,6 +5,8 @@
 import pytest
 from webtest import TestApp
 
+import tests.common.factories
```

Comment:
The other copies of the `factories` fixture do `from ..common import factories`. This won't work in `tests/functional/` because it's not a package, and also aren't relative imports bad? The others also put the import inside the function, which I guess was to avoid a name conflict between the imported module and the fixture function itself? Just importing the module as `tests.common.factories` while verbose avoids the conflict.